packages/compass-data-modeling/src/components/drawer/use-change-on-blur.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useLayoutEffect } from 'react';
+
+export function useChangeOnBlur(
```

Comment:
@Anemy depending on which PR lands first, I think you might want to either copy this over or update your branch to use it